libs/blocks/aside/aside.css

```diff
@@ -747,6 +794,21 @@
     height: var(--icon-size-xxl);
     border-radius: 50%;
   }
+
+  .aside.promobar .foreground.container .promo-text .content-area .text-area {
+    display: flex;
+    flex-flow: column nowrap;
+    gap: var(--spacing-xxs);
+  }
+
+  .aside.promobar .foreground.container .promo-text.mobile-only,
+  .aside.promobar .foreground.container .promo-text.desktop-only {
+    display: none;
+  }
```

Comment:
@JasonHowellSlavin I have made the suggested changes

Reply:
mobile-only tablet-only ans desktop-only are defined in the different media sizes in CSS so that we can show only the viewport specific div and hide the others